qemu/qemu.go

```diff
@@ -768,6 +768,17 @@ type Knobs struct {
 	// Daemonize will turn the qemu process into a daemon
 	Daemonize bool
 
+	// Both HugePages and MemPrealloc require the Memory.Size of the VM
+	// to be set, as they need to reserve the memory upfront in order
+	// for the VM to boot without errors.
+
+	// HugePages always results in memory pre-allocation.
+	// However the setup is different from normal pre-allocation.
+	// Hence HugePages has precedence over MemPrealloc
+
```

Comment:
This newline needs to go as well.

Reply:
@markdryan should be fixed. Sorry for the thrash.